input/tangy-partial-date.js

```diff
@@ -505,7 +505,7 @@ class TangyPartialDate extends PolymerElement {
   }
 
   getValueAsMoment() {
-    this._tranformValueToMoment(this.value)
+    return this._tranformValueToMoment(this.value)
```

Comment:
@rjsteinert If this is used as a helper function on inputs in Tangerine, it should return a value!